config.json.example

```diff
@@ -15,6 +15,9 @@
       "SPIN_ALL_FORTS": true,
       "STAY_WITHIN_PROXIMITY": 9999,
       "AUTO_USE_LUCKY_EGG": false,
+      "CAPTURE": {
+        "MIN_FAILED_ATTEMPTS_BEFORE_USING_BERRY": 1
```

Comment:
maybe make it higher, 1 is too low, since we usually go higher on 30 percent'  3 chances is good. Because if we didn't get in 3 chances, then we need a berry;